### DISCUSSION TITLE
VSA EState descriptor unit

### DISCUSSION THREAD

**Discussion by pratik994** [rdkit/rdkit#5053] [Q&A] [answered]
What is the unit for van der Waal's surface area here? 

rdkit.Chem.EState.EState_VSA.EState_VSA10(mol)[¶](https://www.rdkit.org/docs/source/rdkit.Chem.EState.EState_VSA.html#rdkit.Chem.EState.EState_VSA.EState_VSA10)
EState VSA Descriptor 10 ( 9.17 <= x < 15.00)

**Comment by greglandrum** [comment, marked as answer]
Hi @pratik994 : it's probably best to assume that all of those `EState_VSA` descriptors are effectively unitless

**Reply by pratik994** [reply]
Thank you for the response.
